src/util/config.cpp

```diff
@@ -110,6 +110,12 @@ bool configt::set(const cmdlinet &cmdline)
   if(cmdline.isset("function"))
     main = cmdline.getval("function");
 
+  if(cmdline.isset("class"))
+    cname = cmdline.getval("class");
+
+  if(cmdline.isset("contract"))
+    cname = cmdline.getval("contract");
```

Comment:
Basically, I replaced the old `contract` option with `sol` and added the new `contract` for setting the contract name.

Reply:
So this `cmdline.isset("contract")` can never happen, yes?

Reply:
I think it will happen. Basically, the "contract" is the "class" in solidity. So in solidity, instead of setting `--class xx`, we set `--contract xx`. This is how it will happen

Reply:
Sorry, my mistake, didn't read the changes to the options carefully enough.